### DISCUSSION TITLE
How to start up EJBCA after installation

### DISCUSSION THREAD

**Discussion by Edelmeyder** [Keyfactor/ejbca-ce#66] [Q&A] [answered]
Hello! I'm sorry for this incredibly basic question but I'm absolutely new to all this. I installed EJBCA Community on a linux laptop following the quick-start guide. I got to the point where I can access the admin page on localhost and played arround with that a bit. Now a rebooted the laptop and I don't know how to start it up again. I tried running the bin/ejbca.sh but that tells me "Could not contact EJBCA instance". Thanks in advance!

**Comment by primetomas** [comment, marked as answer]
It's all about starting WildFly. EJBCA is an application inside WildFly you could say. So after a reboot, or shut down of WildFly just restart it with something like:
cd wildfly
bin/standalone.sh

You can configure your system to automatically start wildfly on bootup. There should be some information about that in the documentation somewhere.

Cheers,
Tomas

**Comment by Dan33l** [comment]
@Edelmeyder you should get information step by step here : https://doc.primekey.com/ejbca/ejbca-installation/application-servers/wildfly-24#WildFly24-ConfigureWildFlyasaService
